storybookjs/storybook
# NextJS: Allow disabling next/image lazy loading

## What I did

`next/image` lazy loads images by default. When using Storybook (and especially Chromatic) we usually want images below the fold to load as well so they show up in snapshots. This pull request sets `next/image`'s `loading` property to `eager` which disables lazy loading.

## How to test

1. Create a story using `next/image` and place an image below the fold
2. Open Storybook in your browser
3. Access the story and make sure the image gets loaded even if it's below the fold (or use Chromatic to snapshot the story and make sure images below the fold are loaded and visible)

## Checklist

      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
